test/runtests.jl

```diff
@@ -126,6 +126,13 @@ end
         @test Gray24(0.8)*0.5 === Gray(0.4)
         @test Gray24(0.8)/2   === Gray(0.5f0*N0f8(0.8))
         @test Gray24(0.8)/2.0 === Gray(0.4)
+
```

Comment:
The failed `std` tests in https://github.com/JuliaImages/Images.jl/pull/879 need to move here. But still, leave a copy there so that Images.jl behaves the same to end-users.

Could you also add some tests for `var`? There should be but looks like we didn't do it in Images.jl